lib/mayaUsd/ufe/Global.cpp

```diff
@@ -199,11 +204,13 @@ MStatus initialize()
         = displayInfoMessage;
     dccFunctions.displayMessageFn[static_cast<int>(UsdUfe::MessageType::kWarning)]
         = displayWarningMessage;
-    dccFunctions.displayMessageFn[static_cast<int>(UsdUfe::MessageType::KError)]
+    dccFunctions.displayMessageFn[static_cast<int>(UsdUfe::MessageType::kError)]
```

Comment:
I also fixed this variable name error (from a previous commit of mine) - lowercase "k"